Sources/CNIOSHA1/update_and_patch_sha1.sh

```diff
@@ -55,7 +55,7 @@ $sed -e 's/u_int\([0-9]\+\)_t/uint\1_t/g'
      -e $'/__FBSDID/c #include "include/CNIOSHA1.h"\\n#include <string.h>\\n#if !defined(bzero)\\n#define bzero(b,l) memset((b), \'\\\\0\', (l))\\n#endif\\n#if !defined(bcopy)\\n#define bcopy(s,d,l) memmove((d), (s), (l))\\n#endif\\n#ifdef __ANDROID__\\n#include <sys/endian.h>\\n#elif __linux__\\n#include <sys/types.h>\\n#endif' \
      -i "$here/c_nio_sha1.c"
      
-$sed -e 's/sanity check/soundness check/g' \
+$sed -e 's/sanit[y]/soundness/g' \
```

Comment:
@fabianfett you could just add this line to the above `sed`. So just add

```
    -e 's/sanit[y]/soundness/g' \
```

into the above sed, then we only need one invocation

Reply:
fixed